angr/analyses/reaching_definitions/atoms.py

```diff
@@ -1,5 +1,10 @@
 
 class Atom:
+    """
+    This class represents elements storing values that can be manipulated by IR instructions.
```

Comment:
Perhaps be more explicit about how an atom represent a data storage location of some sort?

Reply:
From what I understand, `Atom` does not specify any "how" as it is a base class.
Subclasses (`Tmp`, `Register`, `MemoryLocation`, `Parameter`) are the ones having different implementations.

Reply:
yes, but, um, an "atom" is a term from static analysis literature referring to a variable as it corresponds to a storage location, regardless of implementation

Reply:
Then just updating to `This class represents data storage locations manipulated by IR instructions.` would work?

Reply:
Yes, that would work.